engine/installation/linux/ubuntu.md

```diff
@@ -31,7 +31,7 @@ storage drivers. **This applies to all versions of Ubuntu**.
 ```bash
```

Comment:
For the above (`linux-image-extra`), we may want to start considering skipping that step for Ubuntu 16.04 and up. Those versions ship with a modern (4.x) kernel, which supports overlay(2). Now that overlay(2) is moved up in the priority list, it will be automatically selected (older versions would default to `devicemapper`)